app/Http/Controllers/BankIntegrationController.php

```diff
@@ -288,12 +328,20 @@ public function removeAccount(AdminBankIntegrationRequest $request, $acc_id)
      */
     public function getTransactions(AdminBankIntegrationRequest $request)
     {
-        /** @var \App\Models\User $user */
-        $user = auth()->user();
+        /** @var \App\Models\Account $account */
+        $account = auth()->user()->account();
```

Comment:
this should change to

$account = auth()->user()->account;

Reply:
done